net/src/ipv6/mod.rs

```diff
@@ -22,10 +22,7 @@ pub mod addr;
 
 /// An IPv6 header
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Ipv6 {
```

Comment:
This change will conflict with #208 which was just merged I think.  May want to drop it from PR or rebase onto main

Reply:
Rebased, should have done this in the first place.